src/frequenz/channels/_broadcast.py

```diff
@@ -246,7 +246,7 @@ def is_closed(self) -> bool:
         return self._closed
 
     async def close(self) -> None:
-        """Close the Broadcast channel.
+        """Close this Broadcast channel.
```

Comment:
to be consistent with other public methods documentation:
```suggestion
        """Close this channel.
```